Apps/WebClient/src/ClientApp/src/views/pcrTest.vue

```diff
@@ -1074,13 +1074,13 @@ export default class PcrTestView extends Vue {
                             </b-col>
                             <b-col cols="8">
                                 <hg-button
-                                    id="btn-submit"
+                                    id="btn-register-kit"
                                     block
                                     type="submit"
                                     variant="primary"
-                                    data-testid="btn-submit"
+                                    data-testid="btn-register-kit"
```

Comment:
NIT: Generally speaking the reason we use data-tested's is that we don't have to change them when making other changes as this means that associated tests may break.  In this case the associated tests are likely in progress but we should ensure that Princess is aware of this change.